en/docs/portal/transports/add-transport.md

```diff
@@ -8,27 +8,26 @@ To add a new transport, click the **Add New Transport** button at the top right
 
 | Fields               | Description   |
 | -------------------- | ------------- |
-| Transport Name       | Enter a name to identify this transport. If you omit this parameter, the HDT portal will generate a default name in the format of "Target Domain:Target Port|
+| Transport Name       | Enter a name to identify this transport. If you omit this parameter, HDT portal will generate a default name in "Target Domain:Target Port" format. |
 | Version Number       | Version number of the transport configuration in history. |
-| Application Protocol | <li><strong> HTTP </strong>: For HTTP protocol.</li> <li><strong> HTTPS </strong>: For HTTPS protocol with SNI (Server Name Indication). |
+| Application Protocol | <strong> HTTP </strong>: For HTTP protocol. <br> <strong> HTTPS </strong>: For HTTPS protocol with SNI (Server Name Indication). |
 | IP Version           | IP version supported by your application, IPv4, IPv6, or both. |
 
 ## Settings
 ![null](</docs/resources/images/transports/add-transport-settings-1.png>)
-![null](</docs/resources/images/transports/add-transport-settings-2.png>)
 
 | Fields               | Description   |
 | -------------------- | ------------- |
-| Fixed Shields        | The shield servers (the HDT servers that connection to your origin servers) will be fixed. |
-| Dedicated IP Service | The IP addresses for serving the transport is dedicated to the CNAME only. The option will NOT be applicable if you select HTTP or HTTPS as the Application Protocol. To use <strong> Dedicated IP Service </strong> for your HTTP/HTTPS applications, select Others as the Application Protocol.|
+| Fixed Shields        | If select Yes, the shield servers (HDT servers that connect to your origin servers) will be fixed, and you can choose the region of shield servers to be fixed in the drop-down list. |
+| Dedicated IP Service | If select Yes, the IP addresses for serving the transport will be dedicated to the CNAME only. The service will NOT be applicable if Application Protocol is HTTP or HTTPS. To use Dedicated IP Service for your HTTP/HTTPS applications, select "Others" as Application Protocol in Basic Info configuration.|
```

Comment:
Please change "If select Yes" to "If you select Yes".

Reply:
The “Others” protocols haven't been supported by NG portal creation, probably we should hide this option when creating transports in NG portal, thanks for pointing this out.

Reply:
Removed the "Dedicated IP Service" option in the latest version.